docs/onboarding.rst

```diff
@@ -36,11 +36,11 @@ The Happy Case
 
     Happy case of a Raiden Transfer
 
-Let's first look at the messages exchanged in the happy path of a Raiden Transfer. Each Transfer has an ``Initiator``, a ``Target`` and zero or more ``Mediators``. The ``Initiator`` creates a `Locked Transfer <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L1124>`_ message and propagates it to the ``Target`` through multiple Mediators.
+Let's first look at the messages exchanged in the happy path of a Raiden Transfer. Each Transfer has an ``Initiator``, a ``Target`` and zero or more ``Mediators``. The ``Initiator`` creates a `Locked Transfer <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L1124>`_ message and propagates it to the ``Target`` through multiple Mediators. A ``LockedTransfer`` message reserves the amount for the pending payment in each channel between ``Initiator``/``Mediator`` and ``Mediator``/``Target``.
 
-Once the ``LockedTransfer`` reaches the ``Target`` then they request the secret from the ``Initiator`` by sending a `Secret Request <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L468>`_ message.
+Once the ``LockedTransfer`` reaches the ``Target`` then they request the secret from the ``Initiator`` by sending a `Secret Request <https://github.com/raiden-network/raiden/blob/38971b372dafb3205cbd3df8cfc3306922a55eac/raiden/messages.py#L468>`_ message.  When the ``Initiator`` receives the ``SecretRequest`` message, they sure that all intermediate channels have the required amount reserved for the pending transfer.
```

Comment:
`they sure that all intermediate` -> `they are sure`
maybe: 
```
he is sure that all intermediate channels have the required amount reserved for the pending transfer and that the mediated transfer has arrived to the ``Target``
```

Reply:
I used your sentence.  Just added a word "locked" before "transfer".